src/h5web/providers/mock/data.ts

```diff
@@ -87,7 +98,7 @@ export function makeDataset(
   return { id, collection: HDF5Collection.Datasets, type, shape, attributes };
 }
 
-function makeSimpleDataset(
+export function makeSimpleDataset(
```

Comment:
Some of the previous mock data functions are still used in the tests of `buildTree`. I thought it unwise to remove the ones that are no longer used anywhere for the time being, in case we want to write more tests or something. Either way, it also didn't want to clutter the diff, so I'm just exporting them to avoid the ESLint warning.